core/logic/src/main/java/org/apache/syncope/core/logic/init/ClassPathScanImplementationLookup.java

```diff
@@ -226,5 +233,15 @@ public void load() {
 
         return passwordRuleClasses.get(passwordRuleConfClass);
     }
+    
+    /**
+     * @return basePackages for syncope class classpath scanning
+     */
+    protected List<String> getBasePackages() {
```

Comment:
Did not get what this method was for, at a first glance, then I've realized it might be used for subclasses: please mention this in the javadoc.